src/main/java/me/zeroeightsix/kami/util/graphics/font/TextComponent.kt

```diff
@@ -1,17 +1,15 @@
-package me.zeroeightsix.kami.util.graphics
+package me.zeroeightsix.kami.util.graphics.font
 
 import me.zeroeightsix.kami.util.Wrapper
 import me.zeroeightsix.kami.util.color.ColorHolder
 import me.zeroeightsix.kami.util.math.Vec2d
-import net.minecraft.client.gui.FontRenderer
 import org.lwjgl.opengl.GL11.*
 import kotlin.math.max
-import kotlin.math.round
 
 /**
  * Renders multi line text easily
  */
-class TextComponent(val separator: String = " ") {
+class TextComponent(val separator: String = "  ") {
```

Comment:
why is there two spaces?

Reply:
TTF font has slightly narrower space. Adding it so it has correct spacing between texts. For example. player name and hp in Nametags